app/packages/app/src/useWriters/onSetFieldVisibilityStage.ts

```diff
@@ -0,0 +1,28 @@
+import {
```

Comment:
I think this writer file can be removed. `useSetters/onSetFieldVisibilityStage` implements everything we need.

Reply:
@benjaminpkane why? When atoms changes, we just need to implement the changes on `useSetters`? What parts need  a writer file?

Reply:
Discussed offline some, but `useSetters` are for changes attached to the page query (e.g. a schema change). That transition happens asynchronously and has a pending state.

`useWriters` register synchronous changes that do not need a page transition. e.g. selecting a sample, we can update the recoil value immediately and dispatch the mutation as a side effect.